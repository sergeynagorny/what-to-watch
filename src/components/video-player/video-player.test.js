import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player";

const film = {
  id: 1,
  title: `What We Do in the Shadows`,
  poster: `img/what-we-do-in-the-shadows.jpg`,
};

it(`Should VideoPlayer render correctly`, () => {
  const tree = renderer.create(
      <VideoPlayer film={film}/>,
      {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
