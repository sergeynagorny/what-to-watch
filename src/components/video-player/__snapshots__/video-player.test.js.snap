// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should VideoPlayer render correctly 1`] = `
<video
  height="175"
  poster="img/what-we-do-in-the-shadows.jpg"
  width="280"
>
  <source />
</video>
`;
